src/plugins/python/python.qbs

```diff
@@ -1,56 +1,40 @@
 import qbs 1.0
 import qbs.Probes as Probes
 import qbs.File
+import qbs.Environment
 
 TiledPlugin {
     Depends { name: "Qt"; submodules: ["widgets"] }
 
     condition: {
-        if (qbs.targetOS.contains("linux")) {
-            return pkgConfig.found;
-        } else if (qbs.targetOS.contains("windows")) {
-            // On Windows, currently only the default install location of
-            // Python 2.7 is supported, and only when compiling with MinGW in
-            // release mode. Also, avoid Python 2.7.10, since it results in a
-            // linker error.
-            //
-            return File.exists("C:/Python27") &&
-                    qbs.toolchain.contains("mingw") &&
-                    !qbs.debugInformation;
-        } else if (qbs.targetOS.contains("darwin")) {
-            return true;
+        if (qbs.targetOS.contains("windows")) {
+            return File.exists( Environment.getEnv("PYTHONHOME") );
         }
 
-        return false;
+        return pkgConfig.found;
     }
 
     Probes.PkgConfigProbe {
         id: pkgConfig
-        name: "python-2.7"
+        // the default qbs.sysroot basically just messes up system defaults
+        sysroot: ''
```

Comment:
Perhaps the reason it defaults to a toolchain path instead of systemwide pkgconfigs is to support cross-compiling? Could perhaps just remove the slightly passive aggressive comment :)

Reply:
qbs manual says of sysroot "This property is typically set in a profile for cross-compiling. Default: Undefined",  my profile was created just by `qbs setup-qt`, not sure what to report yet.